cobigen-setup/index.asciidoc

```diff
@@ -0,0 +1,68 @@
+= Tutorial for Cobigen Setup
+====
+CobiGen is a generic incremental code generator. It allows you to build Java CRUD application based on the devonfw architecture including all software layers. You can generate all necessary classes and services (DAOs, Transfer Objects, simple CRUD use cases with REST services) of the application from one single entity class.
+
+Here in this tutorial we will learn how we can set up the cobigen environment in our system along with eclipse and maven integration. 
+
+Cobigen provides eclipse and maven integration. There is command line interface(CLI) through which we can proceed with integration.
+
+More information about CobiGen on https://devonfw.com/website/pages/docs/master-cobigen.asciidoc.html
+====
+== Prerequisites
+First we need to have devonfw Ide installed in our system to get started with cobigen setup.
+
+[step]
+--
+restoreDevonfwIde(["java","mvn"], "2020.08.001")
+--
+
+Now, devonfw IDE installation is completed.
+Now we will proceed with Cobigen installation.
+[step]
+--
+installCobiGen()
+--
+====
+Below are the commands which we can use on cobigen cli.
+Using the following command and option you will be able to customize your generation as follows
+[step]
+--
+nextKatacodaStep("Commands and Options", [{ "file": "files/commandsAndOptions.md" }])
+--
+
+In order to execute the commands we will have to initialize the cobigen cli first.
+[step]
+--
+nextKatacodaStep("CLI Execution steps", [{ "file": "files/cliExecutionCommands.md" }])
+--
+====

```

Comment:
The `====` here have to be removed. Otherwise the second step (CLI Execution steps) in the block is only rendered as text in the first step and not as own step.